### DISCUSSION TITLE
Problema Alexa dopo passaggio a Clouflare

### DISCUSSION THREAD

**Discussion by AleDestroyer** [indomus/forum#3190] [GENERALE - Discussioni generiche]
Buongiorno,
stamani dopo l'ennesimo problema con duckdns ho migrato su cloudflare.
Ho integrato alexa tramite la vostra guida creando una skill in modalità dev, volevo aggiornare anche li il nuovo indirizzo per raggiungere homeassistant, ma accedendo alla developer console di aws, non danno più la possibilità di editare il codice, è solo un mio problema? Grazie

**Comment by snakuzzo** [comment]
Ma guarda che stai sbagliando tu qualcosa.
Devi semplicemente editare il config.json e cambiare il FQDN. Salvare e deployare.
Ho appena provato e funziona tutto come da anni ormai.

Vai qui...
https://eu-west-1.console.aws.amazon.com/lambda/home?region=eu-west-1#/functions

Selezioni la function e apri il file config.json

**Comment by AleDestroyer** [comment]
Grazie mille per l'aiuto, in effetti è stato un mio errore.

**Comment by AleDestroyer** [comment]
Risolto il problema iniziale, ora se eseguo il test da lamba va a buon fine ma quando attivo la skill, non trova alcun dispositivo e per quelli configurati in precedenza mi dice server offline. Qualche idea?
Ho provato a rimuovere la skill ed a pulire la cache dell'app alexa ma riassociando la skill non trova più i dispositivi di home assistant
